app.lua

```diff
@@ -108,9 +108,13 @@ app:before_filter(function (self)
     if self.params.username then
         self.params.username = self.params.username:lower()
     end
+    self.user = Users:find(self.params.username)
```

Comment:
I think the names are a bit confusing, I can see myself mistaking a user for the other. We could go for something more explicit like `self.visitor` and `self.request_user`, or `self.logged_user` and `self.queried_user`.

Reply:
You're not wrong. At least, I used to feel that way. I looked at a few of leafo's apps, and some other Rails and a couple node apps, and `current_user` and `user` tend to be (I think) the most common combo. `current_user` is very often the common name for the logged in user. However, I've also worked on a (Rails) app for a few years that uses the same convention and so far I think it's worked out ok. 

I have a fair preference for `current_user` to be the currently logged in user, but I do think this could be improved. I like `request_user`, I think. Visitor makes me thing of a guest user, or someone not logged in. 

How's `current_user` and `request_user`? (`current_user` and `queried_user` also I think could work.)

Reply:
Yeah. One gets accustomed to any convention, but we can improve on this one :)

`current_user` works, and I think I slightly prefer `queried_user` because it conveys that it's in the query. `request_user` could be the one doing the request or the one being requested.

What do you think?

Reply:
Yeah, I think that works fine! 👍